libraries/Wire/Wire.h

```diff
@@ -3,14 +3,15 @@
 
 #include <Arduino.h>
 #include "cyhal_i2c.h"
+#include "RingBuffer.h"
 
 class TwoWire {
```

Comment:
By the way, this should be inheriting from HardwareI2C to ensure we are implementing the ArduinoCore-API interface, or?